mcos/optimizer.py

```diff
@@ -102,7 +102,7 @@ def allocate(self, mu: np.array, cov: np.array) -> np.array:
         cov = pd.DataFrame(cov)
 
         if mu is not None:
-            mu = pd.Series(mu)
+            mu = pd.Series(mu.flatten())
```

Comment:
`pd.Series` needs data to be one dimensional so that's why it is flattened.

Reply:
Got it. Can you then also insert a check that asserts that the length of `mu` is the same as one of the dimensions of `cov`?

Reply:
Should this check be above this line? i.e.
```
if mu is not None:
    assertation
    pd.Series(mu.flatten())
```